reference/functions/findfiles.markdown

```diff
@@ -17,15 +17,21 @@ match all files in one directory that end in `.cf` but it won't search
 across directories.  `*/*.cf` on the other hand will look two levels
 deep.
 * `?` matches a single letter
-* `[a-z]` matches any letter from `a` to `z` (not yet supported on Windows)
+* `[a-z]` matches any letter from `a` to `z`
 
 This function, used together with the `bundlesmatching` function,
 allows you to do dynamic inputs and a dynamic bundle call chain.
 
+**WARNING:**
+- The current implementation of glob patterns on Windows contains bugs.
+  Therefore, we strongly advise using the `!windows::` class guard expression to
```

Comment:
Since `advise against` is such a common phrase, that's what I read. I'd write `reccomend` instead:

```suggestion
  Therefore, we strongly recommend using the `!windows::` class guard expression to
```